target/linux/ath79/dts/ar7242_ubnt_sw.dtsi

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar7242.dtsi"
+
+/ {
+	compatible = "ubnt,sw", "qca,ar7242";
+	model = "Ubiquiti Networks SW (rev 1.0) board";
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 12 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
```

Comment:
Can we really expect that all ubnt SW devices share this key/GPIO?

Reply:
Probably yes. Ubiquiti did that on a lot of devices, I'd expect the SW devices to be no different